packages/core/echo/echo-schema/src/formats/types.ts

```diff
@@ -64,6 +64,7 @@ export enum FormatEnum {
   JSON = 'json',
   Markdown = 'markdown',
   Regex = 'regex',
+  SingleSelect = 'single-select',
```

Comment:
Change to option? Or OneOf to match JSON?
This is a format because it maps to a JSON def right?

Reply:
I was thinking we'd have `single-select` and `multi-select`, but I'm happy to change them to `oneOf` and `anyOf`. The select versions seem like the right domain level name to me.